packages/argo-run/src/webpack-config.ts

```diff
@@ -98,23 +97,8 @@ export function createWebpackConfiguration({
             },
           },
         },
-        {
-          test: /\.esnext$/,
-          use: {
-            loader: 'babel-loader',
-            options: {
-              configFile: false,
-              ...babelConfig({
-                react: false,
```

Comment:
There's a difference: `react: false` => `react: useReact`  Is that difference OK?

Reply:
good catch, it should be false for node_modules as we don't wanna compile jsx there, I think.

That'd also go with what @lemonmade suggested to have a different config for node_modules vs app code.